packages/sdk-metrics/test/export/InMemoryMetricExporter.test.ts

```diff
@@ -113,6 +113,24 @@ describe('InMemoryMetricExporter', () => {
     await metricReader.shutdown();
   });
 
+  it('should get no metrics after shutdown', async () => {
+    const counter = meter.createCounter('counter_total', {
+      description: 'a test description',
+    });
+    const counterAttribute = { key1: 'attributeValue1' };
+    counter.add(10, counterAttribute);
+
+    const exportedMetrics = await waitForNumberOfExports(exporter, 1);
+    assert.ok(exportedMetrics.length > 0);
+
+    await exporter.shutdown();
+
+    const otherMetrics = exporter.getMetrics();
+    assert.ok(otherMetrics.length === 0);
```

Comment:
Thanks. I have fixed it.
[use strictEqual function instead of ok](https://github.com/open-telemetry/opentelemetry-js/pull/5214/commits/fd7a69b6efbe61222c8fc49efbb1be2ef7b605d5)

Other `assert.ok(xxx === yyy)` expressions exists in this test file. Should I fix them too??